daemon/cluster/convert/service.go

```diff
@@ -272,7 +272,7 @@ func restartPolicyToGRPC(p *types.RestartPolicy) (*swarmapi.RestartPolicy, error
 	var rp *swarmapi.RestartPolicy
 	if p != nil {
 		rp = &swarmapi.RestartPolicy{}
-		sanatizedCondition := strings.ToUpper(strings.Replace(string(p.Condition), "-", "_", -1))
+		sanatizedCondition := strings.ToUpper(p.Condition.String())
```

Comment:
Perhaps (if I'm correct) adding a short comment to explain this would be good though